src/UniversalDashboard/Themes/Default.psd1

```diff
@@ -83,12 +85,13 @@ $AlternativeBackgroundColor3 = $BackgroundColorBright
             BackgroundColor = $AlternativeBackgroundColor3
             Height = 'auto'
             BoxShadow = 'unset'
+            FontFamily = $FontFamily
 
         }
 
```

Comment:
Had to Set FontFamily in two places to make this stick - I thought doing it on UDDashboard would be enough but nope.